site/.eslintrc.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   ],
   // add your custom rules here
   rules: {
-  }
+  },
+  plugins: ['simple-import-sort'],
```

Comment:
Is this why my import sorting wasn't working!? 😱

Reply:
Maybe :P